tests/unit-tests/test_construct_graph.py

```diff
@@ -116,7 +117,102 @@ def test_label():
     assert np.sum(res['val_mask']) == 1
     assert np.sum(res['test_mask']) == 1
 
+def test_id_map():
```

Comment:
Might be better to break out individual tests instead of testing all functionality in one test unit.